pipelines/binary-builder.yml

```diff
@@ -101,41 +97,41 @@ resources: #####################################################################
   - name: docker-cflinuxfs2-rootfs
     type: docker-image
     source:
-      repository: cloudfoundry/cflinuxfs2
+      repository: jandubois/opensusefs2
```

Comment:
Done with d3711b52331d8a5bb9636f6adaf8f7a8d2e16947, although I'm leaving the pipeline deployed to `concourse.van` using `jandubois/opensusefs2` until the official one is ready.